creation/web_base/glidein_startup.sh

```diff
@@ -1026,7 +1026,8 @@ set_proxy_fullpath() {
 
 [ -n "${X509_USER_PROXY}" ] && set_proxy_fullpath
 
-for tk in $(pwd)/*idtoken; do
+for tk in $(pwd)/credential_*.idtoken; do
+  echo "Setting GLIDEIN_CONDOR_TOKEN to ${tk} " 1>&2
   export GLIDEIN_CONDOR_TOKEN="${tk}"
```

Comment:
Since you are using a for you should protect against there being more than one file matching